### PR TITLE
No unused variables

### PR DESCRIPTION
**Pull request by matteofigus**
Removed some 💩 by setting no unused var setting in jshint and then correcting all the stuff.

### CODE REVIEW

File: src/cli/domain/get-missing-deps.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = function(dependencies, components){
         delete require.cache[pathToModule];
       }
 
-      var required = require(pathToModule);
+      require(pathToModule);
```
Comment (mattiaerre) [inline src/cli/domain/get-missing-deps.js]: I do not understand what this piece of code does 😊

File: src/registry/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var async = require('async');
-var colors = require('colors');
+var colors = require('colors/safe');
```
Comment (mattiaerre) [inline src/registry/index.js]: new version of `colors`?

File: tasks/support/.jshintrc [hunk 3]
```diff
@@ -17,7 +17,7 @@
     "plusplus"      : false,    // true: Prohibit use of `++` & `--`
     "quotmark"      : "single",     // Quotation mark consistency:
     "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
-    "unused"        : false,    // true: Require all defined variables be used
+    "unused"        : true,    // true: Require all defined variables be used
```
Comment (mattiaerre) [inline tasks/support/.jshintrc]: me ❤️  it

File: src/registry/index.js (hunk 2 above)
Comment (matteofigus) [inline src/registry/index.js]: colors exposes the classic mode, which just overrides global strings by allowing to do `"string".red` or the "safe" mode `colors.red("string")`. We use the safe mode almost everywhere but we missed this before..

File: src/cli/domain/get-missing-deps.js (hunk 1 above)
Comment (matteofigus) [inline src/cli/domain/get-missing-deps.js]: The easiest way to know if you can require something is to wrap a require into a try/catch. Given the only value of this is to ensure this doesn't throw, no need to assign the value to a "required" variable (which, in fact, the linter finds as unused)

**Comment by mattiaerre** [comment]
@matteofigus thanks, shall I merge this?

**Pull request merged**